system-monitor/env.yml

```diff
@@ -0,0 +1,31 @@
+name: env
+channels:
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - ca-certificates=2021.1.19=h06a4308_1
+  - certifi=2020.12.5=py37h06a4308_0
+  - ld_impl_linux-64=2.33.1=h53a641e_7
+  - libedit=3.1.20191231=h14c3975_1
+  - libffi=3.3=he6710b0_2
+  - libgcc-ng=9.1.0=hdf63c60_0
+  - libstdcxx-ng=9.1.0=hdf63c60_0
+  - ncurses=6.2=he6710b0_1
+  - openssl=1.1.1j=h27cfd23_0
+  - pip=21.0.1=py37h06a4308_0
+  - python=3.7.10=hdb3f193_0
+  - readline=8.1=h27cfd23_0
+  - setuptools=52.0.0=py37h06a4308_0
+  - sqlite=3.33.0=h62c20be_0
+  - tk=8.6.10=hbc83047_0
+  - wheel=0.36.2=pyhd3eb1b0_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7b6447c_3
+  - pip:
+    - click==7.1.2
+    - flask==1.1.2
+    - itsdangerous==1.1.0
+    - jinja2==2.11.3
+    - markupsafe==1.1.1
+    - werkzeug==1.0.1
+prefix: /home/hubert/miniconda3/envs/env
```

Comment:
This is not universal `prefix`. AFAIK it could be removed and it's still possible to create an env from this file